src/Compilers/CSharp/Test/Symbol/Symbols/ConversionTests.cs

```diff
@@ -1986,6 +1986,138 @@ static void Main()
                 Diagnostic(ErrorCode.ERR_NoImplicitConv, "0").WithArguments("int", "R"));
         }
 
+        [Fact]
+        [WorkItem(434957, "https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_workitems?id=434957")]
+        public void SkipUserDefinedConversionsForThisArgument()
```

Comment:
Just to confirm: this test (and the following 2 or three) already produced errors (just different error code) before your change?